### ISSUE TITLE
IOFactory error when used in Uberjar

### ISSUE THREAD

**Issue by jkutner** [drewr/postal#66]
When postal is used in a uberjar, it fails trying to [read the `pom.properties` file](https://github.com/drewr/postal/blob/master/src/postal/support.clj#L61) with this error:

```
java.lang.IllegalArgumentException: No implementation of method: :make-input-stream of protocol: #'clojure.java.io/IOFactory found for class: nil
```

Stack trace:
https://gist.github.com/kirillsalykin/011e5479d0f2b9c2a81d

**WORKAROUND**:
Add `-Dpostal.version=1.11.3` to java process

Here is a reproducible example:
https://github.com/InspireNL/TeamHappiness

See stackoverflow for discussion:
http://stackoverflow.com/questions/32235680/sending-email-with-postal-from-heroku-fires-strange-exception/

**Comment by martinklepsch** [comment]
I also experienced this issue.

**Comment by martinklepsch** [comment]
@drewr any chance we can get a patch release if a PR is provided?

**Comment by drewr** [comment]
Yes! PR welcome. The packaging is very old and needs to be redone. I use a `version.txt` file in `resources/` in other projects which works with uberjars too. Will do it if you don't get to something like that first. Thanks!

**Comment by danielsz** [comment]
Experienced the same issue.
